incremental-font-transfer/src/glyph_keyed.rs

```diff
@@ -273,15 +284,31 @@ impl WritableOffset for u32 {
     }
 }
 
+impl WritableOffset for Uint24 {
```

Comment:
Maybe it would be worth taking a look at the overall design for how we do serialization in write-fonts? I'm wondering if it would be simpler for you to have some kind of buffer type (`BeBuffer` is another ref here) and then it would be trivial to have a method there like [`TableData::write`](https://github.com/googlefonts/fontations/blob/bfe1c5554e5281afdaa14f2b1447fe4034a0b69f/write-fonts/src/write.rs#L218), which would let you write any scalar in place.

Reply:
Actually just realizing I can use the klippa serializer here too like I do during table reconstruction. I'll make that update and let's see how it looks.